# Medidas de Tendencia Central y Percentiles

## Objetivo:

Aplicar las medidas de tendencia central (media, mediana y moda) y medidas de posición no central (percentiles) en el análisis de datos, utilizando Python como herramienta de apoyo.

*Nota: Este laboratorio está diseñado para desarrollar razonamiento estadístico. Evita el uso de herramientas de IA para responder las preguntas a menos que esto sea explícitamente sugerido* 🤖. *En su lugar, enfócate en aplicar los conceptos revisados.*

## Actividad 1

La siguiente base de datos posee los registros de todas las calificaciones obtenidas por los alumnos del segundo semestre del 2019, jornada diurna en la asignatura de Álgebra.

Para responder las siguientes preguntas, descarga la base de datos **<a href="https://drive.google.com/uc?export=download&id=10tm66MrnBpkR1_aY10yxEAudwirzDcJK
">aquí</a>**.

1. ¿Cuál es la población en estudio?

La población corresponde a los alumnos de álgebra de segundo semestre año 2019, jornada diurna.

2. ¿Cuál es la muestra en estudio?

In [2]:
import pandas as pd
df_4 = pd.read_csv('Actividad_1.csv')

In [3]:
muestra_4 = df_4.shape[0]
muestra_4

516

La muestra corresponde a 516 alumnos de álgebra de segundo semestre año 2019, jornada diurna.

3.  Identifique y clasifique las variables involucradas en el estudio.

In [4]:
df_4.head()

ID Alumno  Taller 1  Taller 2  Taller 3  Actividad en línea 1  \
0  160001322       5.5       7.0       5.6                   6.4   
1  160002042       7.0       7.0       7.0                   3.8   
2  160002581       6.5       6.2       4.5                   5.4   
3  160001794       7.0       7.0       6.5                   7.0   
4  160003667       6.0       4.3       6.0                   4.8   

   Actividad en línea 2  Actividad en línea 3  Prueba 1  Prueba 2  Prueba 3  \
0                   6.2                   4.6       6.2       5.5       4.3   
1                   2.3                   2.3       5.0       4.9       4.4   
2                   1.8                   2.4       6.2       5.1       4.0   
3                   5.3                   2.6       6.2       7.0       6.3   
4                   4.4                   3.0       5.3       4.7       4.2   

   Nota Presentación  Nota Examen  Nota Final  
0                5.5          6.2         5.8  
1                4.8          5.2         5.0  
2                5.0          4.7         4.9  
3                6.4          6.6         6.5  
4                4.7          5.8         5.1

Las variables involucradas en el estudio son: talleres, actividades en línea, pruebas, nota de presentación, nota de examen y nota final.

La clasificación de todas las variables es: Cuantitativa continua.

4. Calcule e interprete las medidas de tendencia central de las notas del examen.

In [5]:
media_examen = df_4['Nota Examen'].mean()
mediana_examen = df_4['Nota Examen'].median()
moda_examen = df_4['Nota Examen'].mode()

print(f'Promedio: {media_examen:.1f}\nMediana: {mediana_examen}\nModa: {moda_examen[0]}')

Promedio: 5.1
Mediana: 5.2
Moda: 5.3


* Media: El promedio de las notas del examen el año 2019 fue de 5,1 en la asignatura de álgebra, jornada diurna.
* Mediana: El 50% de las notas del examen el año 2019 fue de a lo más 5,2 en la asignatura de álgebra, jornada diurna.
* Moda: La nota más frecuente en el examen de álgebra del año 2019, jornada diurna, fue de 5,3.

5. ¿Entre qué valores se encuentra el 60% de las notas más bajas en la prueba 1?

In [6]:
#Valor máximo del 60% más bajo
p_60 = df_4['Prueba 1'].quantile(0.6)

#Valor mínimo del 60% más bajo
minimo_prueba1 = df_4['Prueba 1'].min()

print(f'Percentil 60: {p_60}\nMínimo: {minimo_prueba1}')

Percentil 60: 5.7
Mínimo: 1.6


El 60% de las notas más bajas en la prueba 1 está entre 1,6 y 5,7.

6.  La institución de educación creará cursos de reforzamiento para todos los estudiantes cuyas calificaciones en la Prueba 2, estén en el 25% más bajo. ¿Cuál es la nota más alta dentro de este grupo que va a reforzamiento?

In [7]:
p25_prueba2 = df_4['Prueba 2'].quantile(0.25)
print(p25_prueba2)

4.4


La nota más alta del grupo es 4,4.

7. ¿Cuál es la nota más baja entre el 30% de mejores notas del examen?

In [8]:
p70_ex = df_4['Nota Examen'].quantile(0.7)
print(p70_ex)

5.7


La nota más baja, entre el 30% de mejores notas del examen, es un 5,7.

8. Si se desea reevaluar al 10% más bajo en el examen, ¿cuál es rango de notas que se debe considerar?

In [9]:
#Valor máximo del 10% más bajo
p10_ex = df_4['Nota Examen'].quantile(0.1)

#Valor mínimo del 10% más bajo
minimo_ex = df_4['Nota Examen'].min()

print(f'Percentil 10: {p10_ex}\nMínimo: {minimo_ex}')

Percentil 10: 3.5
Mínimo: 1.3


Se debe considerar a los alumnos que obtuvieron entre 1,3 y 3,5 en el examen.

9. Si la institución va a premiar al 5% de las mejores notas finales, ¿cuál es la nota más baja registrada por ese grupo?

In [10]:
p95_final = df_4['Nota Final'].quantile(0.95)
print(p95_final)

6.5


La nota más baja registrada por ese grupo es un 6,5.

## Actividad 2

Imagina que trabajas como analista de datos en una consultora de orientación vocacional que asesora a estudiantes de enseñanza media sobre qué carrera elegir. Recibes una base de datos con información de carreras técnicas y profesionales de distintas instituciones chilenas de los años 2024 y 2025. Tu misión es analizar los ingresos y empleabilidad de los titulados, y entregar conclusiones claras que los ayuden a tomar decisiones.

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. Identifica la población y muestra del estudio.

In [12]:
import pandas as pd
df_1 = pd.read_excel('Actividad_2_3_4.xlsx')

In [13]:
muestra_1 = df_1.shape[0]
muestra_1

1163

La población corresponde a carreras dictadas en Chile durante los años 2024 y 2025.

La muestra corresponde a 1.163 carreras dictadas en Chile durante los años 2024 y 2025.

2. Clasifica las variables presentes en la base de datos.

In [14]:
#Mostraremos las primeras cinco filas del DF
df_1.head()

Código           Tipo de institución  Acreditación institución (años)  \
0     456  Centros de Formación Técnica                                5   
1     456  Centros de Formación Técnica                                5   
2     456  Centros de Formación Técnica                                5   
3     456  Centros de Formación Técnica                                5   
4     456  Centros de Formación Técnica                                5   

               Área                            Nombre carrera genérica  \
0        Tecnología  Técnico en Electricidad y Electricidad Industrial   
1             Salud                              Técnico en Enfermería   
2  Ciencias Básicas                              Técnico en Geominería   
3        Tecnología                Técnico en Mantenimiento Industrial   
4        Tecnología                     Técnico en Mecánica Industrial   

  Duración Real (semestres)  Empleabilidad 1er año (%)  \
0                  6.537975                  76.616915   
1                  6.280899                  62.500000   
2                  7.914286                  61.627907   
3                  6.927461                  72.000000   
4                  6.659794                  82.122905   

   Ingreso al 4° año (pesos)  
0                    1150000  
1                     750000  
2                    1150000  
3                    1150000  
4                    1150000

La clasificación de las variables es:

* Tipo de institución: cualitativa nominal.
* Acreditación institución (años): cuantitativa discreta.
* Área: cualitativa nominal.
* Nombre carrera genérica: cualitativa nominal.
* Duración Real (semestres): cuantitativa continua.
* Empleabilidad al 1er año (%): cuantitativa continua.
* Ingreso al 4to año (pesos): cuantitativa continua.

3. ¿Cuál es el ingreso promedio de los titulados? ¿Y la mediana? ¿Qué puedes concluir de la diferencia entre ambos?

In [15]:
promedio_todos = df_1['Ingreso al 4° año (pesos)'].mean()
mediana_todos = df_1['Ingreso al 4° año (pesos)'].median()

print(f'Promedio: ${promedio_todos:.0f}\nMediana: ${mediana_todos}.')

Promedio: $1307481
Mediana: $1150000.0.


El ingreso promedio al 4to año de egreso de todos los titulados es de \$1.307.481. La mediana es de \$1.150.000, esto indica que el 50% de los titulados tiene ese ingreso o menos al 4to año de egreso.

La diferencia entre la media y la mediana indica que hay titulados con ingresos mucho más altos que el promedio del grupo, lo cual eleva el promedio. Esto muestra que la media es una medida de tendencia que se ve afectada por valores extremos altos (o bajos) por lo que no siempre es un buen representante de todos los datos.

4. ¿Cuál es la moda de los ingresos? Interprétala.

In [16]:
moda_ingreso = df_1['Ingreso al 4° año (pesos)'].mode()
moda_ingreso

0    1050000
Name: Ingreso al 4° año (pesos), dtype: int64

El ingreso más frecuente al 4to año de egreso es \$ 1.050.000.

5. Calcula los percentiles 25, 50 y 75 del ingreso de los titulados e interprétalos.

In [17]:
p_25_ingresos = df_1['Ingreso al 4° año (pesos)'].quantile(0.25)
p_50_ingresos = df_1['Ingreso al 4° año (pesos)'].quantile(0.5)
p_75_ingresos = df_1['Ingreso al 4° año (pesos)'].quantile(0.75)
print(f'Percentil 25: {p_25_ingresos}\nPercentil 50: {p_50_ingresos}\nPercentil 75: {p_75_ingresos}')

Percentil 25: 950000.0
Percentil 50: 1150000.0
Percentil 75: 1550000.0


* El 25% de los titulados tiene un ingreso al 4to año de egreso de a lo más \$950.000.

* El 50% de los titulados tiene un ingreso al 4to año de egreso de a lo más \$1.150.000.

* El 75% de los titulados tiene un ingreso al 4to año de egreso de a lo más \$1.550.000.

6. Utiliza la función *describe()* de Pandas para validar tus resultados.

La función *describe()* de Pandas, permite calcular los tres percentiles pedidos en el inciso anterior de todas las variables cuantitativas de la base de datos. También permite realizar cálculos de otras medidas estadísticas como el promedio, el mínimo, máximo, entre otras.

In [18]:
df_1.describe().astype(int)

Código  Acreditación institución (años)  Empleabilidad 1er año (%)  \
count    1163                             1163                       1163   
mean      102                                5                         67   
std       103                                1                         18   
min         1                                0                         10   
25%        50                                5                         54   
50%        84                                5                         69   
75%       111                                6                         81   
max       701                                7                        100   

       Ingreso al 4° año (pesos)  
count                       1163  
mean                     1307480  
std                       506349  
min                       550000  
25%                       950000  
50%                      1150000  
75%                      1550000  
max                      3000000

7. Considera que tres personas desean estudiar Técnico en enfermería, pero cada una valora cosas distintas:
* Sofía busca maximizar su ingreso futuro.
* Diego prioriza encontrar trabajo rápidamente.
* Camila quiere un equilibrio entre ambas.

$\hspace{1cm}$Considerando la media como medida representativa, ¿qué tipo de institución recomendarías a cada uno y por qué?

In [19]:
#Vamos a filtrar por Técnico en Enfermería
tecnico_enfermeria = df_1[df_1['Nombre carrera genérica'] == 'Técnico en Enfermería']

#Agruparemos por tipo de institución y calcularemos el promedio de la
#empleabilidad y del ingreso
tabla_1 = tecnico_enfermeria.groupby('Tipo de institución')[
    ['Empleabilidad 1er año (%)', 'Ingreso al 4° año (pesos)']].mean()

round(tabla_1,1)

Empleabilidad 1er año (%)  \
Tipo de institución                                       
Centros de Formación Técnica                       64.0   
Insitutos Profesionales                            66.8   
Universidades                                      65.0   

                              Ingreso al 4° año (pesos)  
Tipo de institución                                      
Centros de Formación Técnica                   740909.1  
Insitutos Profesionales                        750000.0  
Universidades                                  790000.0

Sofía, que busca maximizar su ingreso futuro, debería optar por estudiar Técnico en Enfermería en una universidad, ya que es el tipo de institución que reporta el ingreso promedio más alto al 4° año ($790.000).

Diego, que prioriza encontrar trabajo rápidamente, debería considerar un instituto profesional (IP), que muestra la mayor empleabilidad al 1er año (66,8%), ligeramente superior al resto.

Camila, que busca un equilibrio entre ingreso y empleabilidad, podría elegir una universidad ya que, el sueldo es superior a las otras instituciones (cerca de un 5\% superior a un IP) y su empleabilidad es ligeramente menor a un IP (aproximadamente un 2,7\%), logrando un buen balance entre ambos factores.

## Actividad 3

Un grupo de estudiantes está evaluando si el nivel de acreditación de una institución podría impactar sus resultados laborales. Para responder esta pregunta, realiza el siguiente análisis:

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. Agrupa los datos por años de acreditación y calcula la media y la mediana del ingreso al 4to año de cada grupo.

In [20]:
import pandas as pd
df_2 = pd.read_excel('Actividad_2_3_4.xlsx')
df_2.head(3)

Código           Tipo de institución  Acreditación institución (años)  \
0     456  Centros de Formación Técnica                                5   
1     456  Centros de Formación Técnica                                5   
2     456  Centros de Formación Técnica                                5   

               Área                            Nombre carrera genérica  \
0        Tecnología  Técnico en Electricidad y Electricidad Industrial   
1             Salud                              Técnico en Enfermería   
2  Ciencias Básicas                              Técnico en Geominería   

  Duración Real (semestres)  Empleabilidad 1er año (%)  \
0                  6.537975                  76.616915   
1                  6.280899                  62.500000   
2                  7.914286                  61.627907   

   Ingreso al 4° año (pesos)  
0                    1150000  
1                     750000  
2                    1150000

In [ ]:
tabla_2 = df_2.groupby('Acreditación institución (años)')[
    ['Empleabilidad 1er año (%)', 'Ingreso al 4° año (pesos)']].agg(['mean','median'])

round(tabla_2,1)

Empleabilidad 1er año (%)         \
                                                     mean median   
Acreditación institución (años)                                    
0                                                    63.9   69.4   
3                                                    69.3   69.0   
4                                                    63.3   65.0   
5                                                    63.6   65.6   
6                                                    71.3   74.0   
7                                                    71.7   74.3   

                                Ingreso al 4° año (pesos)             
                                                     mean     median  
Acreditación institución (años)                                       
0                                               1024666.7   950000.0  
3                                               1313043.5  1150000.0  
4                                               1153020.1  1050000.0  
5                                               1192729.6  1050000.0  
6                                               1541260.2  1450000.0  
7                                               1421043.2  1250000.0

2. ¿Existe una relación aparente entre la cantidad de años de acreditación y el ingreso promedio? ¿Qué patrón observas?

Se observa que en general, a medida que aumentan los años de acreditación también aumenta el ingreso promedio, desde \$1.024.667 con 0 años de acreditación hasta \$1.541.260 con 6 años. Esto muestra un patrón de mejora en los ingresos asociado a instituciones con más años de acreditación.

3. ¿Se observa una tendencia similar al analizar la empleabilidad al primer año?

Al analizar la relación entre la media de la empleabilidad al 1er año y la acreditación, también se observa una tendencia al alza, pasando de 63,9% en instituciones sin acreditación hasta 71,7% en instituciones con 7 años.

4. Comparando las variables ingreso y empleabilidad, analiza cómo varían la media y la mediana en cada caso. ¿Qué conclusiones generales puedes extraer sobre la representatividad de estas medidas?

Comparando ambas variables, en el caso del ingreso la media excede a la mediana en todos los niveles, lo que indica que hay titulados con ingresos muy altos que elevan el promedio. En cambio, para la empleabilidad la mediana supera a la media en la mayoría de los casos, lo que indica que la mayoría de las instituciones alcanza tasas de colocación por encima del promedio. Por lo tanto, para describir la empleabilidad es más útil usar la mediana, mientras que en el análisis de ingresos conviene considerar la media junto con la mediana para entender la influencia de valores extremos.

5. Teniendo en cuenta tu análisis sobre cómo varían los ingresos y la empleabilidad con los años de acreditación, ¿qué consejo le darías a un estudiante que va a elegir una institución basándose únicamente en el nivel de acreditación? Justifica tu recomendación.

Es recomendable optar por instituciones con al menos 6 años de acreditación, ya que en ese tramo se observan los mejores resultados. El ingreso promedio al 4° año de egreso supera ampliamente el millón de pesos y la empleabilidad promedio al 1er año alcanza el 71,3 %.

## Actividad 4

Imagina que trabajas como analista del Ministerio de Educación. Te han solicitado caracterizar los resultados laborales de los titulados por área de formación utilizando una base de datos con información de carreras técnicas y profesionales de distintas instituciones chilenas de los años 2024 y 2025.

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. ¿Cuál Área se observa con mayor frecuencia entre los titulados?

In [21]:
import pandas as pd
df_3 = pd.read_excel('Actividad_2_3_4.xlsx')
df_3.head(2)

Código           Tipo de institución  Acreditación institución (años)  \
0     456  Centros de Formación Técnica                                5   
1     456  Centros de Formación Técnica                                5   

         Área                            Nombre carrera genérica  \
0  Tecnología  Técnico en Electricidad y Electricidad Industrial   
1       Salud                              Técnico en Enfermería   

  Duración Real (semestres)  Empleabilidad 1er año (%)  \
0                  6.537975                  76.616915   
1                  6.280899                  62.500000   

   Ingreso al 4° año (pesos)  
0                    1150000  
1                     750000

In [22]:
moda_area = df_3['Área'].mode()
moda_area[0]

'Tecnología'

El área que se observa con mayor frecuencia entre los titulados es Tecnología.

2. ¿Entre qué sueldos se encuentran los egresados del 10% más alto al 4to año de titulación?

In [23]:
#Valor mínimo del 10% más alto
p_90 = df_3['Ingreso al 4° año (pesos)'].quantile(0.9)

#Valor máximo del 10% más alto
maximo = df_3['Ingreso al 4° año (pesos)'].max()

print(f'Percentil 90: {p_90}\nMáximo: {maximo}')

Percentil 90: 1950000.0
Máximo: 3000000


Los egresados con los sueldos en el 10% más alto, ganan entre \$1.950.000 y \$3.000.000 al 4to año de titulación.

3. ¿Entre qué sueldos se encuentran los egresados del 17% más bajo al 4to año de titulación?

In [ ]:
#Valor máximo del 17% más bajo
p_17 = df_3['Ingreso al 4° año (pesos)'].quantile(0.17)

#Valor mínimo del 17% más bajo
minimo = df_3['Ingreso al 4° año (pesos)'].min()

print(f'Percentil 17: {p_17}\nMínimo: {minimo}')

Percentil 17: 950000.0
Mínimo: 550000


Los egresados con los sueldos en el 17% más bajo, ganan entre \$550.000 y \$950.000 al 4to año de titulación.

4. Agrupa los datos por Área y para cada grupo calcula las siguientes medidas del Ingreso al 4° año:
* Media
* Mediana
* Percentil 25
* Percentil 75

$\hspace{1cm}$Luego, construye un DataFrame que reúna estas cuatro columnas.

In [24]:
#Agruparemos los datos por Área
datos_area = df_3.groupby('Área')['Ingreso al 4° año (pesos)']

#Calculamos las medidas solicitadas
media_area = datos_area.mean()
mediana_area = datos_area.median()
p25_area = datos_area.quantile(0.25)
p75_area = datos_area.quantile(0.75)

#Incluimos la información en un nuevo DF
tabla_3 = pd.DataFrame({
    'Media': media_area,
    'Mediana': mediana_area,
    'Percentil 25': p25_area,
    'Percentil 75': p75_area
})

round(tabla_3, 0)

Media    Mediana  Percentil 25  Percentil 75
Área                                                                       
Administración y Comercio  1221930.0  1150000.0      950000.0     1450000.0
Agropecuaria               1176471.0  1250000.0      975000.0     1325000.0
Arte y Arquitectura        1079825.0  1050000.0      850000.0     1350000.0
Ciencias Básicas           1575000.0  1400000.0     1125000.0     2025000.0
Ciencias Sociales          1155042.0  1150000.0     1050000.0     1250000.0
Derecho                    1561111.0  1550000.0     1400000.0     1800000.0
Educación                   963990.0   950000.0      950000.0     1050000.0
Humanidades                 870000.0   850000.0      850000.0      850000.0
Salud                      1454510.0  1350000.0      950000.0     1750000.0
Tecnología                 1552448.0  1450000.0     1050000.0     1925000.0

5. Interpreta las medidas calculadas en las áreas de Agropecuaria y Tecnología.

Las interpretaciones para el área de Agropecuaria son:
* Media: Los egresados del área Agropecuaria reciben un ingreso promedio de \$1.176.471 al 4to año de titulación.
* Mediana: El 50% de los egresados del área Agropecuaria recibe un ingreso de \$1.250.000 o menos, al 4to año de titulación.
* Percentil 25: El 25% de los egresados del área de Agropecuaria tienen un ingreso de \$975.000 o menos, al 4to año de titulación.
* Percentil 75: El 75% de los egresados del área de Agropecuaria tienen un ingreso de \$1.450.000 o menos, al 4to año de titulación.

Las interpretaciones para el área de Tecnología son:
* Media: Los egresados del área Tecnología reciben un ingreso promedio de \$1.552.448 al 4to año de titulación.
* Mediana: El 50% de los egresados del área Tecnología recibe un ingreso de \$1.450.000 o menos, al 4to año de titulación.
* Percentil 25: El 25% de los egresados del área de Tecnología tienen un ingreso de \$1.050.000 o menos, al 4to año de titulación.
* Percentil 75: El 75% de los egresados del área de Tecnología tienen un ingreso de \$1.925.000 o menos, al 4to año de titulación.

6. Al comparar la media con la mediana en el Área Salud se observa que la media es mayor. ¿Qué característica de los datos podría explicar esta diferencia?

En el área Salud la media (\$1.454.510) es mayor que la mediana (\$1.350.000) porque existen ingresos altos (por ejemplo, sobre el percentil 75 = \$1.750.000) que elevan el promedio. Estos valores extremos "arrastran" la media hacia arriba, mientras que la mediana no se ve afectada por ellos.

7. Describe lo que ocurre con la media del ingreso al 4to año de titulación, según Área y Tipo de institución. ¿Qué conclusiones puedes extraer acerca del impacto del tipo de institución en el ingreso según área?

*Sugerencia: Puedes crear una tabla bivariada utilizando la función crosstab() de Pandas*.

In [25]:
tabla_4 = pd.crosstab(df_3['Área'], df_3['Tipo de institución'],
                      df_3['Ingreso al 4° año (pesos)'],
                      aggfunc = 'mean', margins = True, margins_name= 'Total')
round(tabla_4,0)

Tipo de institución        Centros de Formación Técnica  \
Área                                                      
Administración y Comercio                      895455.0   
Agropecuaria                                   870000.0   
Arte y Arquitectura                            750000.0   
Ciencias Básicas                              1050000.0   
Ciencias Sociales                              683333.0   
Derecho                                        850000.0   
Educación                                      608333.0   
Humanidades                                         NaN   
Salud                                          728261.0   
Tecnología                                    1084375.0   
Total                                          878431.0   

Tipo de institución        Insitutos Profesionales  Universidades      Total  
Área                                                                          
Administración y Comercio                1038525.0      1430682.0  1221930.0  
Agropecuaria                              907143.0      1331818.0  1176471.0  
Arte y Arquitectura                       854167.0      1259375.0  1079825.0  
Ciencias Básicas                         1150000.0      1795455.0  1575000.0  
Ciencias Sociales                        1035714.0      1185294.0  1155042.0  
Derecho                                   850000.0      1618000.0  1561111.0  
Educación                                 673810.0      1028750.0   963990.0  
Humanidades                               900000.0       850000.0   870000.0  
Salud                                     780435.0      1608612.0  1454510.0  
Tecnología                               1110241.0      1854678.0  1552448.0  
Total                                     982773.0      1454557.0  1307481.0

En todas las áreas los titulados de universidades tienen un mayor ingreso promedio, mientras que los titulados de CFT registran valores más bajos. Los institutos profesionales se encuentran en un punto medio en casi todas las áreas. La brecha salarial es especialmente grande en Tecnología, Salud, Derecho y Ciencias básicas, donde la diferencia entre universidad y CFT supera con creces a la observada en otras áreas.